js/modules/k6/execution/execution.go

```diff
@@ -208,3 +219,81 @@ func newInfoObj(rt *goja.Runtime, props map[string]func() interface{}) (*goja.Ob
 
 	return o, nil
 }
+
+type tagsDynamicObject struct {
+	Runtime *goja.Runtime
+	State   *lib.State
+}
+
+// Get a property value for the key. May return nil if the property does not exist.
+func (o *tagsDynamicObject) Get(key string) goja.Value {
+	tag, ok := o.State.Tags.Get(key)
+	if !ok {
+		return nil
+	}
+	return o.Runtime.ToValue(tag)
+}
+
+// Set a property value for the key. It returns true if succeed.
+// String, Boolean and Number types are implicitly converted
+// to the goja's relative string representation.
+// In any other case, if the Throw option is set then an error is raised
+// otherwise just a Warning is written.
+func (o *tagsDynamicObject) Set(key string, val goja.Value) bool {
+	switch val.ExportType().Kind() { //nolint:exhaustive
+	case
+		reflect.String,
+		reflect.Bool,
+		reflect.Int,
+		reflect.Int8,
+		reflect.Int16,
+		reflect.Int32,
+		reflect.Int64,
+		reflect.Uint,
+		reflect.Uint8,
+		reflect.Uint16,
+		reflect.Uint32,
+		reflect.Uint64,
+		reflect.Float32,
+		reflect.Float64,
+		reflect.Complex64,
+		reflect.Complex128:
```

Comment:
```suggestion
		reflect.String,
		reflect.Bool,
		reflect.Int64,
		reflect.Float64,
```
I am [pretty sure goja doesn't use any of the others](https://github.com/dop251/goja/blob/dc8c55024d06009a0cba080cd738192d368735de/value.go#L85-L87), but cc @MStoykov WDYT?

Reply:
I already applied the change because the linked goja's code makes the suggestion reasonable. I will wait for the @MStoykov's opinion before rebasing.

Reply:
Given [this list](https://github.com/dop251/goja/blob/dc8c55024d06009a0cba080cd738192d368735de/value.go#L44) of predefined reflect types that then get returned by `ExportType` it seems like this should be enough. 

I had some suspicion that maybe if it goes down some rabbit hole of reflecting go objects it might return the original int type but testing with `struct{X int8}` still gives you `int64` from my quick testing, so I guess that isn't a problem as well